spec/system/candidate_interface/continuous_applications/withdraws/candidate_withdraws_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.feature 'A candidate withdraws their application', :bullet do
```

Comment:
I don't think I see an unhappy path scenario (click continue, see errors) -- would be nice to include one (if I've missed it ignore this!)

Reply:
I don't think you can get errors if you click `continue` it's basically a link_to to the feedback form.

You can get errors on the feedback form though, so I'll put in a test for that